### PR TITLE
Update build_tutorial.md

### PR DESCRIPTION
**Pull request by Connoropolous**
This piece of documentation is out of date going back to the time before `holochain/holochain` was public, and tryorama rsm was in a branch of public tryorama repo so this PR brings it back in date for now.

also fix other broken links

This piece of documentation is linked to from the interim RSM docs overview on developer.holochain.org so it can actually get traffic. 
https://developer.holochain.org/holochain-rsm-guidance

### CODE REVIEW

File: docs/build_tutorial.md [hunk 1]
```diff
@@ -1,38 +1,79 @@
...
+
+Install Rust
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Switch to the proper Rust "toolchain" that Holochain needs
+```bash
+rustup install stable-2020-08-03
+rustup default stable-2020-08-03
```
Comment (freesig) [inline docs/build_tutorial.md:56]: Is there any reason to pin the rust version in this doc? Won't it just go out of date? We should be able to always support the latest stable and if not that's a bug

**Review by freesig — APPROVED** [review]
Nice work!
I think we don't need to pin the rust version though :slightly_smiling_face:

**Comment by Connoropolous** [comment]
Well, it won't compile if you just use latest, or anything beyond 1.45.2 aka the release I dropped in

File: docs/build_tutorial.md (hunk 1 above)
Comment (Connoropolous) [reply docs/build_tutorial.md:56]: This note in the `core_testing.md` doc provides the reference that demonstrates why you can't use latest stable :s
https://github.com/holochain/holochain/blob/develop/docs/core_testing.md#requirements

**Comment by freesig** [comment]
ah I see #329 Let me have a go

**Comment by Connoropolous** [comment]
"We should fix the bug vs putting the workaround in the docs is what I mean." I support that notion 👍 haha
I will remove it from this doc in anticipation of such a fix

**Pull request merged**